src/index.local.ts

```diff
@@ -1,18 +1,31 @@
 import 'core-js';
+
 import * as https from 'https';
 import * as pem from 'pem';
-import { ExpressServer } from './app';
+import { LoggerInstance } from 'winston';
+import { CommonConfig } from './config/CommonConfig';
+import { ExpressServer } from './server/ExpressServer';
 import { INJECTOR } from './util/di/Injector';
+import { Logger } from './util/di/LoggerFactory';
 
 const expressServer = INJECTOR.get(ExpressServer) as ExpressServer;
+const config = INJECTOR.get(CommonConfig) as CommonConfig;
+const logger = INJECTOR.get(Logger) as LoggerInstance;
+
+const port = config.PORT;
+
+if (!config.USE_HTTP) {
```

Comment:
add an env var  USE_HTTP  for start the server with http only. Which is debug friendly.